docs/run_publish/references.md

```diff
@@ -100,6 +105,42 @@ This displays the current version.
 0.19.1
 ```
 
+### reindex
+
+- In order to use this command you need to have `@subql/node` v1.10.0 or above.
+
+When using reindex command, historical must be enabled for the targeted project(`--disable-historical=false`).
+
+Use `--targetHeight=<blockNumber>` with `reindex` to remove indexed data and reindex from specified block height.
```

Comment:
Why do they need to use a `targetHeight` command?

Reply:
reindex is an subcommand, rather than a flag, hence to use yargs to pass in its values a flag should be used